web-ui/src/components/reports-section/CheckinReport.jsx

```diff
@@ -1,36 +1,188 @@
-import React, { useContext } from 'react';
+import React, { useContext, useState } from 'react';
 import { Link } from 'react-router-dom';
+import PropTypes from 'prop-types';
 
 import { getAvatarURL } from '../../api/api.js';
 import { AppContext } from '../../context/AppContext';
 import { selectFilteredCheckinsForTeamMemberAndPDL } from '../../context/selectors';
 
+import ExpandMoreIcon from '@mui/icons-material/ExpandMore';
+import ExpandMore from '../expand-more/ExpandMore';
+
 import {
   Accordion,
   AccordionDetails,
   AccordionSummary,
   Avatar,
   Box,
   Card,
-  CardHeader,
   CardContent,
+  CardHeader,
   Chip,
+  Collapse,
   Container,
+  Divider,
+  Stepper,
+  Step,
+  StepLabel,
   Typography
 } from '@mui/material';
 
+/**
+ * @typedef {Object} Checkin
+ * @property {string} id - The ID of the check-in.
+ * @property {boolean} completed - Indicates whether the check-in is completed.
+ * @property {Array} checkinDate - The date of the check-in.
+ * @property {string} pdlId - The ID of the PDL.
+ * @property {string} teamMemberId - The ID of the team member.
+ */
+
+/**
+ * @typedef {("Done" | "In Progress" | "Not Started")} CheckinStatus
+ * @typedef {("Not Yet Scheduled" | "Scheduled" | "Completed")} SchedulingStatus
+ */
+
+/** @type {CheckinStatus} */
```

Comment:
Shouldn't this type be `CheckinStatus[]` so it's an array?

Reply:
The `CheckinStatus` type is one of several strings, and not an array type.

Reply:
Good eyes. Thanks, Mark!